workflow.rst

```diff
@@ -134,8 +134,8 @@ like this:
                         'enabled' => true
                     ],
                     'marking_store' => [
-                        'type' => 'multiple_state', // or 'single_state'
```

Comment:
@noniagriconomie I am wondering if there is no need for changing the XML config 🤔 in this example...

Reply:
@OskarStark what dou you mean? i fear not understanding you

Reply:
I meant, you update the YAML, and PHP example, but not the XML config in between...

Reply:
understood, will definitly patch it

Reply:
Thank you 👍🏻